External/FEXCore/Source/Interface/Core/JIT/x86_64/JIT.cpp

```diff
@@ -293,7 +294,8 @@ void X86JITCore::Op_Unhandled(IR::IROp_Header *IROp, IR::NodeID Node) {
       case FABI_UNKNOWN:
       default:
 #if defined(ASSERTIONS_ENABLED) && ASSERTIONS_ENABLED
-        LOGMAN_MSG_A_FMT("Unhandled IR Fallback ABI: {} {}", FEXCore::IR::GetName(IROp->Op), Info.ABI);
+        LOGMAN_MSG_A_FMT("Unhandled IR Fallback ABI: {} {}",
```

Comment:
Shouldn't this be `ERROR_AND_DIE`?

Reply:
All the other backends seem to use LOGMAN_MSG_A_FMT for similar scenarios, so I just left it as is